index.html

```diff
@@ -153,13 +143,10 @@ <h2>Multi-Post Stories</h2>
       <li><a href=""><img src="./img/Rectangle70.png"alt="rectangular"></a></li>
       <li><a href="">Ruby</a></li> 
     </ul>
-  </section4>
+
   
-  <section5>
     <!-- SEE PROJECT BUTTON -->
-    <div class="bottom">
-      <input class="btn btn-primary" type="submit" value="See project">
-    </div>
+    <button class="btn">See project</button>
```

Comment:
- [ Optional ] As stated in the project requirement I recommend you to Implement the button interactions  follow the button styling: Active, Hover and Pressed (see design).
![Capture](https://user-images.githubusercontent.com/93598189/192723678-03c45faf-6dcf-479d-94c1-eaca26b0bd5a.PNG)